### PR TITLE
add support for citations attribute

### PR DESCRIPTION
**Pull request by sahiljhawar**
Solves #362
Uses a Mixin base class with `citation` attribute to be inherited by other lightcurve classes.

Example usage:

```python
kn = SVDLightCurveModel(model="Bu2019lm",sample_times=[0,0.5],svd_path="path/to/svd",local_only=True,filters=["2massh"])
grb  = GRBLightCurveModel(sample_times=[0,0.5])

combined = GenericCombineLightCurveModel(models=[kn,grb],sample_times=[0,0.5])

combined.citation
>>> [{'Bu2019lm': ['https://arxiv.org/abs/2002.11355',
    'https://arxiv.org/abs/1906.04205']},
    {'TrPi2018': ['https://arxiv.org/abs/1909.11691']}]

kn.citation
>>>{'Bu2019lm': ['https://arxiv.org/abs/2002.11355',
   'https://arxiv.org/abs/1906.04205']}

grb.citation
>>>{'TrPi2018': ['https://arxiv.org/abs/1909.11691']}
``` 

Few citations are missing

### CODE REVIEW

File: nmma/em/model.py [hunk 1]
```diff
@@ -99,12 +99,52 @@
     ],
 }
 
-
-class GenericCombineLightCurveModel(object):
+class Mixin:
```
Comment (mcoughlin) [inline nmma/em/model.py:102]: This looks good. But could we call this Lightcurve or some other useful name?

**Review by mcoughlin — APPROVED** [review]
I think this looks good, let's just use a better name than Mixin.

File: nmma/em/model.py (hunk 1 above)
Comment (sahiljhawar) [reply nmma/em/model.py:103]: Mixin is a standard name for a such a base class. Any suggestions for other name?

Comment (mcoughlin) [reply nmma/em/model.py:103]: Ah sorry should have been more clear. I meant like LightcurveMixin or similar.

**Review by mcoughlin — APPROVED** [review]
LGTM

**Pull request merged**